src/redux/actions/wallet.js

```diff
@@ -30,4 +34,4 @@ const updateBalance = () => async (dispatch, getState) => {
   }
 }
 
-export default { initialize }
+export default { initialize, updateBalance }
```

Comment:
`updateBalance` doesn't actually need to be exposed AFAIK

Reply:
:boom: